DualView/src/androidTest/java/com/microsoft/device/display/samples/dualview/MapImageTest.kt

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.dualview
+
+import androidx.compose.ui.graphics.asAndroidBitmap
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.captureToImage
+import androidx.compose.ui.test.hasAnySibling
+import androidx.compose.ui.test.hasContentDescription
+import androidx.compose.ui.test.hasParent
+import androidx.compose.ui.test.hasScrollToIndexAction
+import androidx.compose.ui.test.hasText
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performGesture
+import androidx.compose.ui.test.performScrollToIndex
+import androidx.compose.ui.test.swipeDown
+import androidx.window.testing.layout.WindowLayoutInfoPublisherRule
+import com.microsoft.device.display.samples.dualview.models.restaurants
+import com.microsoft.device.display.samples.dualview.ui.theme.DualViewAppTheme
+import com.microsoft.device.display.samples.dualview.ui.view.DualViewApp
+import com.microsoft.device.display.samples.dualview.ui.view.MapView
+import com.microsoft.device.dualscreen.testutils.assertScreenshotMatchesReference
+import com.microsoft.device.dualscreen.testutils.compare
+import com.microsoft.device.dualscreen.testutils.getString
+import com.microsoft.device.dualscreen.testutils.saveScreenshotToDevice
+import com.microsoft.device.dualscreen.testutils.simulateHorizontalFold
+import com.microsoft.device.dualscreen.testutils.zoomIn
+import com.microsoft.device.dualscreen.testutils.zoomOut
+import com.microsoft.device.dualscreen.windowstate.rememberWindowState
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.rules.TestRule
+
+const val VIEW_SIZE = 400
+
+class MapImageTest {
+    private val composeTestRule = createAndroidComposeRule<MainActivity>()
+    private val publisherRule = WindowLayoutInfoPublisherRule()
+
+    @get: Rule
+    val testRule: TestRule
+
+    init {
+        testRule = RuleChain.outerRule(publisherRule).around(composeTestRule)
+        RuleChain.outerRule(composeTestRule)
+    }
+
+    /**
+     * Temporary test to save screenshots of composables to use as reference images - uncomment test
+     * annotation to run and follow instructions below to transfer images from device to computer:
+     * https://stackoverflow.com/questions/40323126/where-do-i-find-the-saved-image-in-android
+     *
+     * You may also need to add write permissions to AndroidManifest.xml in your own project.
+     */
+    @ExperimentalTestApi
+    // @Test
+    fun app_saveScreenshots() {
```

Comment:
Why do we need this?

Reply:
We can take this out! I wasn't sure if it would be helpful for developers to see, but I used the test once to save screenshots of the composables to use as reference images for future tests.